aspnetcore/mvc/controllers/actions.md

```diff
@@ -14,80 +14,91 @@ uid: mvc/controllers/actions
+    This type returns a redirect to an action or destination (using `Redirect`, `LocalRedirect`, `RedirectToAction`, or `RedirectToRoute`). For example, `return RedirectToAction("Complete", new {id = 123});` redirects to the named action method and passes to it an anonymous object.
 
-**HTTP Status Code**
+* **HTTP Status Code**
 
-Return an HTTP status code. Example: `return BadRequest();`
+    This type returns an HTTP status code. A couple helper methods of this type are `BadRequest` and `NotFound`. For example, `return BadRequest();` produces a 400 status code when executed.
 
-**Formatted Response**
+#### 2. A non-empty response body with a predefined content type
 
-Return `Json` or similar to format an object in a specific manner. Example: `return Json(customer);`
+Most of these response types include a `ContentType` property, allowing you to set the `Content-Type` response header to describe the response body.
 
-**Content negotiated response**
+There are two response types within this category: [View](xref:mvc/views/overview) and [Formatted Response](xref:mvc/models/formatting).
 
-Instead of returning an object directly, an action can return a content negotiated response (using `Ok`, `Created`, `CreatedAtRoute` or `CreatedAtAction`). Examples: `return Ok();` or `return CreatedAtRoute("routename",values,newobject);`
+* **View**
 
-**Redirect**
+    This type returns a view which uses a model to render HTML. For example, `return View(customer);` passes a model to the view for data-binding.
 
-Returns a redirect to another action or destination (using `Redirect`, `LocalRedirect`, `RedirectToAction` or `RedirectToRoute`). Example: `return RedirectToAction("Complete", new {id = 123});`
+* **Formatted Response**
 
-In addition to the methods above, an action can also simply return an object. In this case, the object will be formatted based on the client's request. Learn more about [Formatting Response Data](../models/formatting.md)
+    This type returns JSON or a similar data exchange format to represent an object in a specific manner. For example, `return Json(customer);` serializes the provided object into JSON format.
+
+#### 3. A non-empty response body formatted in a content type negotiated with the client
+
+This type is better known as a **Content Negotiated Response**. [Content negotiation](xref:mvc/models/formatting#content-negotiation) applies whenever an action returns an [ObjectResult](https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.aspnetcore.mvc.objectresult) type or something other than an [IActionResult](https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.aspnetcore.mvc.iactionresult) implementation. 
+
+Instead of returning an object directly, an action can return a content negotiated response. Some helper methods of this type include `Created`, `CreatedAtAction`, `CreatedAtRoute`, and `Ok`. Examples: `return Ok();` or `return CreatedAtRoute("routename", values, newobject);`
```

Comment:
No pref, but considering the content of the previous paragraph I think this sentence could just be deleted.